catalog/transformer/catalog-transformer-common/src/main/java/ddf/catalog/transformer/common/tika/MetacardCreator.java

```diff
@@ -68,24 +71,54 @@ public static Metacard createMetacard(final Metadata metadata, final String id,
             metacard.setAttribute(new AttributeImpl(Metacard.MODIFIED, modifiedDate));
         }
 
-        if (StringUtils.isNotBlank(id)) {
-            metacard.setAttribute(new AttributeImpl(Metacard.ID, id));
-        }
-
-        if (StringUtils.isNotBlank(metadataXml)) {
-            metacard.setAttribute(new AttributeImpl(Metacard.METADATA, metadataXml));
-        }
+        setAttribute(metacard, Metacard.ID, id);
 
+        setAttribute(metacard, Metacard.METADATA, metadataXml);
         final String lat = metadata.get(Metadata.LATITUDE);
         final String lon = metadata.get(Metadata.LONGITUDE);
-        final String wkt = toWkt(lon, lat);
-        if (StringUtils.isNotBlank(wkt)) {
-            metacard.setAttribute(new AttributeImpl(Metacard.GEOGRAPHY, wkt));
-        }
+        setAttribute(metacard, Metacard.GEOGRAPHY, toWkt(lon, lat));
+
+        setAttribute(metacard, Media.FORMAT, metadata.get(TikaCoreProperties.FORMAT));
+
+        setAttribute(metacard, Core.DESCRIPTION, metadata.get(TikaCoreProperties.DESCRIPTION));
+
+        setAttribute(metacard, Media.TYPE, metadata.get(Metadata.CONTENT_TYPE));
+
+        setAttribute(metacard, Media.BITS_PER_SAMPLE, metadata.get(TIFF.BITS_PER_SAMPLE));
+
+        setAttribute(metacard, Media.HEIGHT, metadata.get(TIFF.IMAGE_LENGTH));
+
+        setAttribute(metacard, Media.WIDTH, metadata.get(TIFF.IMAGE_WIDTH));
+
+        setAttribute(metacard, Media.COMPRESSION, metadata.get(COMPRESSION_TYPE));
+
+        setAttribute(metacard, Contact.CREATOR_NAME, metadata.get(TikaCoreProperties.CREATOR));
+
+        setAttribute(metacard, Topic.KEYWORD, metadata.get(TikaCoreProperties.KEYWORDS));
```

Comment:
No it does not. Currently it will save the first value from the keyword metadata and set it as it's keyword.